packages/react-button/src/components/CompoundButton/useCompoundButtonStyles.ts

```diff
@@ -91,19 +91,19 @@ const useRootStyles = makeStyles({
 
   // Size variations
   small: {
-    ...shorthands.padding(buttonSpacing.medium),
+    ...shorthands.padding(buttonSpacing.medium, buttonSpacing.medium, '10px', buttonSpacing.medium),
 
     fontSize: tokens.fontSizeBase300,
     lineHeight: tokens.lineHeightBase300,
   },
   medium: {
-    ...shorthands.padding(buttonSpacing.large),
+    ...shorthands.padding('14px', buttonSpacing.large, buttonSpacing.larger, buttonSpacing.large),
```

Comment:
Is there any actual value in using `buttonSpacing` here? It seems a little arbitrary to be mixing `14px`, `large`, `larger` here.. those named sizes don't seem to have any semantic meaning for a button size of `medium`. Maybe these should just all be hardcoded pixel values?

Reply:
They are in the design spec but are not reflected in the theme. I'll replace them for now and have a follow up conversation with design.

Reply:
I didn't necessarily mean to delete `buttonSpacing` entirely. I was just referring to this case where you're mixing the sizes.

It's ok if you want to delete it, but also feel free to restore `buttonSpacing` in the other cases. I'll go ahead and approve this PR either way.